modin/core/dataframe/pandas/dataframe/dataframe.py

```diff
@@ -2108,6 +2339,48 @@ def concat(self, axis, others, how, sort):
             new_partitions, new_index, new_columns, new_lengths, new_widths, new_dtypes
         )
 
+    def groupby(
```

Comment:
again, why override a parent method without actually implementing it?

Reply:
See: https://github.com/modin-project/modin/pull/3717#discussion_r769155299